lib/analytics.js

```diff
@@ -59,6 +59,17 @@ function Analytics() {
 }
 
 /**
+ * Used for stubbing navigator.doNotTrack in the unit tests.
+ */
+
+Analytics._doNotTrack = function() {
+  // Fallback to window.doNotTrack in older browsers
+  var doNotTrack = navigator.doNotTrack || window.doNotTrack;
+  // Normalize values (Firefox can return 'unspecified')
+  return doNotTrack === '1' ? '1' : '0';
```

Comment:
Because I thought it would be a good idea to return the spec values in `context.doNotTrack`.